packages/input-iban/translations/sk.js

```diff
@@ -16,5 +16,20 @@ export default {
       'RO {rumunský}\n' +
       'other {{params}}\n' +
       '} kód {fieldName}.',
+    IsNotCountryIBAN:
+      '{params, select,\n' +
+      'AT {rakúsky}\n' +
```

Comment:
Start of sentence, so with a capital?

Reply:
Hmm yeah in this case we should indeed capitalise, good catch.

Reply:
Pushed as separate commit, I think I did it correctly even with cyrillic haha